package.json

```diff
@@ -98,7 +98,7 @@
     "clean:build": "shx rm -rf .cache",
     "commit": "git-cz",
     "generate": "yarn plop",
-    "lint:md": "yarn eslintpackages/patternfly-4 --ext md --config .eslintrc-md.json",
+    "lint:md": "yarn eslint packages/patternfly-4 --ext md --config .eslintrc-md.json",
```

Comment:
Can we add `lint:md` to be run in `.circleci/config.yml` so we don't accidentally break it again? Also would be nice to always lint our MD every PR.